src/main/java/appeng/integration/abstraction/IJEI.java

```diff
@@ -1,15 +1,25 @@
 package appeng.integration.abstraction;
 
 
+import appeng.integration.IIntegrationModule;
+
+
 /**
  * Abstracts access to the JEI API functionality.
  */
-public interface IJEI
+public interface IJEI extends IIntegrationModule
 {
 
-	boolean isEnabled();
+	default String getSearchText()
+	{
+		return "";
+	}
 
-	String getSearchText();
+	default void setSearchText( String searchText )
+	{
+	}
 
-	void setSearchText( String searchText );
+	class Stub extends IIntegrationModule.Stub implements IJEI
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2176)